camel-knative/camel-knative/src/main/java/org/apache/camel/component/knative/KnativeConfiguration.java

```diff
@@ -31,8 +31,8 @@
     private KnativeEnvironment environment;
     @UriParam
     private String serviceName;
-    @UriParam(defaultValue = "0.3", enums = "0.1,0.2,0.3")
-    private String cloudEventsSpecVersion = CloudEvents.V03.version();
+    @UriParam(defaultValue = "0.1", enums = "0.1,0.2,0.3,1.0")
```

Comment:
Is this correct should default value not be "1.0" ?

Reply:
damn, will fix it

Reply:
done